lib/plugins.js

```diff
@@ -246,6 +246,10 @@ exports.categories = {
         susie: {
             url: 'https://github.com/mtharrison/susie',
             description: 'Server-Sent Events for hapi with support for streaming events'
+        },
+        hapi-wechat: {
```

Comment:
This key needs single quotes (`'`) around it because it contains a dash (`-`).

Reply:
oh, thanks.